apps/backend/src/modules/friends/friend.controller.ts

```diff
@@ -550,6 +557,10 @@ class FriendController extends BaseController {
 	 *     security:
 	 *       - bearerAuth: []
 	 *     parameters:
+	 * 		- name: search
+	 * 		   in: query
```

Comment:
```suggestion
	 * 		- name: search
	 * 		in: query
```
In other places we have such indentation